src/reducers/themeReducer.ts

```diff
@@ -24,3 +28,8 @@ export const themeReducer = (
       return state;
   }
 };
+
+export const themePicker = (theme: Themes) => (dispatch: Dispatch) => {
```

Comment:
this is thunk func and it should not be here, in separate file, here we have only reducer.